### ISSUE TITLE
n + 1 query problem

### ISSUE THREAD

**Issue by developer239** [EQuimper/twitter-clone-with-graphql-reactnative#23]
Could you please mention handling n + 1 query problems? 

![image](https://user-images.githubusercontent.com/14946081/29709041-260fc566-898b-11e7-9382-cc53f8c00d02.png)

**Comment by EQuimper** [comment]
Yes I can check for it, this tutorial for sure is not always best practice. Sorry about that. But I try to do my best here.

**Comment by developer239** [comment]
@EQuimper The tutorial is really good. Especially the graphql part. There are not many useful tutorials showing how to build graphql servers unfortunately.

**Comment by EQuimper** [comment]
Thank you for that @developer239 :)
